SRAM increase in Bluetooth  [samples: bbc_microbit: pong fails to build]

when trying to build samples/boards/bbc_microbit/pong

I get the following error:

```
/opt/zephyr-sdk-0.14.2/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/10.3.0/../../../../arm-zephyr-eabi/bin/ld.bfd: zephyr_pre0.elf section `noinit' will not fit in region `SRAM'
/opt/zephyr-sdk-0.14.2/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/10.3.0/../../../../arm-zephyr-eabi/bin/ld.bfd: region `SRAM' overflowed by 1079 bytes
collect2: error: ld returned 1 exit status
```

Doing a git bisect, tracked it down to:

commit 3c5bf5089ad8465b82e59c57fb1ad8b78b48d971 (refs/bisect/bad)
Author: Jonathan Rico <jonathan.rico@nordicsemi.no>
Date:   Mon May 30 16:08:11 2022 +0200

    Bluetooth: host: add dedicated WQ for long-running tasks

--- a/samples/boards/bbc_microbit/pong/prj.conf
+++ b/samples/boards/bbc_microbit/pong/prj.conf
@@ -9,3 +9,7 @@ CONFIG_DISPLAY=y
 CONFIG_MICROBIT_DISPLAY=y
 CONFIG_PWM=y
 CONFIG_PWM_NRF5_SW=y
+
+# This feature isn't needed as this sample is intended to be used
+# between zephyr devices only.
+CONFIG_BT_GATT_CACHING=n

